Kinvey/KinveyTests/RealtimeAclTestCase.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  RealtimeAclTestCase.swift
+//  Kinvey
+//
+//  Created by Victor Hugo on 2017-08-25.
+//  Copyright © 2017 Kinvey. All rights reserved.
+//
+
+import XCTest
+import Kinvey
+
+class RealtimeAclTestCase: KinveyTestCase {
+    
+    func testLiveStreamAclConsturctorNoParams() {
```

Comment:
@heyzooi there's a typo in the method name :)
Also, it'll be nice to add some more tests around the other parameters in the constructor.

Reply:
ok, I will add more tests specifically for ACL on LiveService

Reply:
as we discussed, I will create new tickets to test ACL on LiveService and merge this PR as is now